GIDs

```diff
@@ -205,7 +205,7 @@ meta1q:*:261:
 meta1c:*:262:meta1s
 meta1m:*:263:meta1s,meta1q
 meta1:*:264:
-# free: 265
+cantemo:*:265:
```

Comment:
I dont like this diverging from FreeBSD, someone may take it meanwhile and make our lives harder.

Perhaps lets try miwi acquire a gid/uid upstream? Meanwhile we could pick a number in the middle to decrease the chance for someone to take it.

Reply:
@miwi-fbsd can you please see if we can acquire a gid/uid upstream ? For now i'll proceed with a number in the middle then